openshift/secret.yaml

```diff
@@ -6,5 +6,8 @@ metadata:
     app: cyborg-regidores
 type: Opaque
 data:
-  kafka-bootstrap-servers: dGhyZWUta2Fma2EtYm9vdHN0cmFwLWRoLXByb2QtbWVzc2FnZS1idXMuY2xvdWQuZGF0YWh1Yi51cHNoaWZ0LnJlZGhhdC5jb206NDQz
-  root-ca-cert: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURKekNDQWcrZ0F3SUJBZ0lKQUxOV0dWRjVDRGdrTUEwR0NTcUdTSWIzRFFFQkN3VUFNQ294RXpBUkJnTlYKQkFvTUNtbHZMbk4wY21sdGVta3hFekFSQmdOVkJBTU1DbU5zZFhOMFpYSXRZMkV3SGhjTk1UZ3hNREkxTVRZdwpPVEU0V2hjTk1Ua3hNREkxTVRZd09URTRXakFxTVJNd0VRWURWUVFLREFwcGJ5NXpkSEpwYlhwcE1STXdFUVlEClZRUUREQXBqYkhWemRHVnlMV05oTUlJQklqQU5CZ2txaGtpRzl3MEJBUUVGQUFPQ0FROEFNSUlCQ2dLQ0FRRUEKcGVWb0RBdnliZWtiamduSkRDM01CbnJydWhUM0ZUcjlOMnNXSjZ6NWR5ekYzVVRVUzNvaGpPRDhNbE9QQjdwNQpPYUNwdVd5OFh6aTdiUHB1K0lwMURMcVZnZGtCWE42V1VyeE5QNk1Pb2F6WHdKQXF1SExzNkN1VS9iSFRUL3FWCm0vK3VPZ3NsTWNCTmFZN2o5NHFmYjhTdnZlY0pNMEg0ZEZ0N1BlU3BXNDJ4TWZxTzNxQStLbFFUQVBmaE9BRVgKSDYrK2hkZnVaenV3MG5URXFmTGhvdkNmOVQyZUIzcEVzNmpYU1AvMk5MeXVRWWRTMVB1NSswZ1dZazNCRVVhbAp4QXJGWnpVcmVDYlA4QWNKdHVEQTczaVB6ZW1jdmx3V2ppWURiM25yNUtGNmgxSjNDckYremt2VkI1aWJEM0hzCk1KTzd5dVNDMEFJSWUxa1NJUVVVSFFJREFRQUJvMUF3VGpBZEJnTlZIUTRFRmdRVXBGSTRMa2llK0NpSklJblAKeWJQaUxNZTk3RU13SHdZRFZSMGpCQmd3Rm9BVXBGSTRMa2llK0NpSklJblB5YlBpTE1lOTdFTXdEQVlEVlIwVApCQVV3QXdFQi96QU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUFFQllHUFRvdmdsY3I2Y3RqNzhvR3R1dFRRM0tvClMwV2NURjJrWlU1ejJjMm5YOWdDOFFBQUpjYTVqUENBdG9WalJTN3JkUU9YaHVvN3ZoZTJTNnMxNjY5ODJSUkkKZWg5WlBleEQzK2JSaVMvMDYyNmZkVGVOa05YRk9YaFpITUpnUEt1M3BQeDZXeThMZWVabmUwem5sRmVRd3hWUQp5SGdQcmRyeC9URk5Hc1NtNUsxaWNkWDFrVDdWZUY5UXdUYnorQy9xQ2RscHZlWDJFc3g1SlZIekM3OXhqUnAxCld5WEFqMTIwZy83bG5pbVVNRm1VTUNNcWMveW9QWE5QenpETTdHQmhvUkVROThVTjcrQXNRazhORXBKWFhNMzUKazVvZTBwT1kzSG5acUhKckFnUm9EUHdxc25KUk9BZ0p1RGlZclV3WkM4YmVyeklNTXNhV25KaWFOdz09Ci0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K
+  kafka-bootstrap-servers: a2Fma2EuZGF0YWh1Yi5yZWRoYXQuY29tOjQ0Mw==
+  root-ca-cert:
```

Comment:
@harshad16 you love checking secrets into git, right?! https://kubernetes.io/docs/concepts/configuration/secret/#risks

Reply:
@goern, updated the pr.
apologizes for messing up the things.